src/main/java/frc/robot/subsystems/arm/ArmCal.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems.arm;
+
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+
+public class ArmCal {
+    public static final double PLACEHOLDER_DOUBLE = 0.0;
+    public static final float PLACEHOLDER_FLOAT = 0;
+    public static final int PLACEHOLDER_INT = 0;
+    
+    public static final double ARM_P = PLACEHOLDER_DOUBLE;
+    public static final double ARM_I = PLACEHOLDER_DOUBLE;
+    public static final double ARM_D = PLACEHOLDER_DOUBLE;
+    public static final double ARM_MAX_VELOCITY_DEG_PER_SECOND = PLACEHOLDER_DOUBLE;
+    public static final double ARM_MAX_ACCELERATION_DEG_PER_SECOND_SQUARED = PLACEHOLDER_DOUBLE;
```

Comment:
These should be formatted to all share a comment (and you should add a comment), like how we've done in the past.